### DISCUSSION TITLE
different syncPolicy per env/app?

### DISCUSSION THREAD

**Discussion by mubarak-j** [argoproj-labs/applicationset#382] [Q&A] [answered]
I’m using Argocd with Applicationset and curious if it’s possible to set syncPolicy to allow auto-sync on one app/env and not the other, e.g allow auto-sync on staging and disable for production?

**Comment by jgwest** [comment, marked as answer]
For the time being, you would need to use multiple ApplicationSets: one for each syncPolicy you want to use, and then ensure that the ApplicationSet generates the subset of Application CRs that you wish to use that syncPolicy (how to do this depends on the generator, for example, using label selectors withe the Cluster generator),

We should get closer to being able to handle this as one ApplicationSet with this feature: https://github.com/argoproj-labs/applicationset/issues/71
